diff --git a/web/modules/contrib/opigno_statistics/src/Services/UserStatisticsManager.php b/web/modules/contrib/opigno_statistics/src/Services/UserStatisticsManager.php
index 4c2a0b46..c9007a1b 100644
--- a/src/Services/UserStatisticsManager.php
+++ b/src/Services/UserStatisticsManager.php
@@ -709,8 +709,8 @@ public function buildTrainingsList(int $uid, bool $completed = FALSE): array {
         ['data' => $details, 'class' => 'details'],
       ];
       if ($completed) {
-        $timestamp = strtotime($row->completed ?: 'now');
-        $date = $this->dateFormatter->format($timestamp, 'day_month_year');
+        $timestamp = strtotime($row->completed ?? 'now');
+        $date = $this->dateFormatter->format($timestamp, 'custom', 'm/d/Y');
         array_splice($table_row, 2, 0, $date);
       }
 
@@ -778,7 +778,7 @@ public function buildCertificatesList(int $uid): array {
       $gid = $row->gid ?? 0;
       $name = $this->t('Certificate for @training', ['@training' => $row->name ?? '']);
       $timestamp = strtotime($row->completed ?? 'now');
-      $date = $this->dateFormatter->format($timestamp, 'day_month_year');
+      $date = $this->dateFormatter->format($timestamp, 'custom', 'm/d/Y');
 
       // Generate the certificate download link.
       $params = ['entity_type' => 'group', 'entity_id' => $gid];
